cirrus/server/tests/test_main.py

```diff
@@ -508,18 +522,67 @@ def test_get_features_with_and_without_nimbus_preview(
             ],
         }
 
-        # Without nimbus_preview
```

Comment:
Each of these cases should really be their own individual test case rather than folded into one.

Reply:
good idea, will split this one